examples/demo-app/src/app.js

```diff
@@ -453,17 +378,6 @@ class App extends Component {
     }
   }
 
-  _loadSampleData() {
-    // this._loadPointData();
```

Comment:
Are these useful in some way? Do we need to keep them for local experiments or can they be deleted?

Reply:
It was already included in keplers source code, so let's keep it to quickly test and verify if we encounter a bug?